resolwe_bio/management/commands/generate_samples.py

```diff
@@ -264,9 +271,9 @@ def create_data(self, reads_name='seq_reads', annotated=False, rseed=None):
             sample.descriptor = self.generate_sample_desciptor(d.name)
             sample.presample = False
             sample.save()
-            print "Created sample: {} (id={})".format(sample.name, sample.id)
+            self.stdout.write("Created sample: {} (id={})".format(sample.name, sample.id))
```

Comment:
I suggest you change this to `logger.info` and check directly in the DB. For instance, you can check if 1 sample was added to the Sample model after you call the command and maybe even check its name. You could check if the sample has the required data objects in the Data model as well. @dblenkus can you comment if this makes sense?